code/modules/antagonists/eldritch_cult/eldritch_items.dm

```diff
@@ -180,3 +189,219 @@
+	name = "void cloak"
+	desc = "Black like tar, doesn't reflect any light. Runic symbols line the outside, with each flash you loose comprehension of what you are seeing."
+	icon_state = "void_cloak"
+	inhand_icon_state = "void_cloak"
+	allowed = list(/obj/item/melee/sickly_blade, /obj/item/forbidden_book, /obj/item/living_heart)
+	hoodtype = /obj/item/clothing/head/hooded/cult_hoodie/void
+	// slightly worse than normal cult robes
+	armor = list(MELEE = 30, BULLET = 30, LASER = 30,ENERGY = 30, BOMB = 15, BIO = 0, RAD = 0, FIRE = 0, ACID = 0)
+	pocket_storage_component_path = /datum/component/storage/concrete/pockets/void_cloak
+
+/obj/item/clothing/suit/hooded/cultrobes/void/ToggleHood()
+	if(!iscarbon(src.loc))
+		return
+	var/mob/living/carbon/carbon_user = src.loc
+	if(IS_HERETIC(carbon_user) || IS_HERETIC_MONSTER(carbon_user))
+		. = ..()
+		//We need to account for the hood shenanigans, and that way we can make sure items always fit, even if one of the slots is used by the fucking hood.
+		var/datum/component/storage/concrete/pockets/void_cloak/cloak = GetComponent(/datum/component/storage/concrete/pockets/void_cloak)
+		if(suittoggled)
+			cloak.max_items = 3
+			to_chat(carbon_user,"<span class='notice'>The light shifts around you making the cloak invisible!</span>")
+		else
+			cloak.max_items = 4
+	else
+		to_chat(carbon_user,"<span class='danger'>You can't force the hood onto your head!</span>")
+
+	item_flags = suittoggled ? EXAMINE_SKIP : ~EXAMINE_SKIP
+
+/obj/item/clothing/mask/void_mask
+	name = "Abyssal Mask"
+	desc = "Mask created from the suffering of existance, you can look down it's eyes, and notice something gazing back at you."
+	icon_state = "mad_mask"
+	inhand_icon_state = "mad_mask"
+	w_class = WEIGHT_CLASS_SMALL
+	flags_cover = MASKCOVERSEYES
+	resistance_flags = FLAMMABLE
+	flags_inv = HIDEFACE|HIDEFACIALHAIR
+	var/mob/living/carbon/human/local_user
+
+/obj/item/clothing/mask/void_mask/equipped(mob/user, slot)
+	. = ..()
+
+	if(ishuman(user) && user.mind && slot == ITEM_SLOT_MASK)
+		local_user = user
+		START_PROCESSING(SSobj,src)
+		if(IS_HERETIC(user) || IS_HERETIC_MONSTER(user))
+			return
+		ADD_TRAIT(src, TRAIT_NODROP, CLOTHING_TRAIT)
+
+/obj/item/clothing/mask/void_mask/dropped(mob/M)
+	. = ..()
+	local_user = null
+	STOP_PROCESSING(SSobj,src)
+	REMOVE_TRAIT(src, TRAIT_NODROP, CLOTHING_TRAIT)
+
+/obj/item/clothing/mask/void_mask/process(delta_time)
+	if(!local_user)
+		return PROCESS_KILL
+
+	if((IS_HERETIC(local_user) || IS_HERETIC_MONSTER(local_user)) && HAS_TRAIT(src,TRAIT_NODROP))
+		REMOVE_TRAIT(src, TRAIT_NODROP, CLOTHING_TRAIT)
+
+	for(var/mob/living/carbon/human/human_in_range in spiral_range(9,local_user))
+		if(IS_HERETIC(human_in_range) || IS_HERETIC_MONSTER(human_in_range))
+			continue
+
+		SEND_SIGNAL(human_in_range,COMSIG_DRAIN_SANITY,rand(-1,-10))
+
+		if(DT_PROB(10,delta_time))
+			human_in_range.emote(pick("giggle","laugh"))
+			human_in_range.adjustStaminaLoss(10)
```

Comment:
unless you stack up like 50 monkeymen wearing these masks into a single tile or something, the stamina damage dealt by this mask is absolute dogshit (remember, stamina damage heals back to full after you go 10 seconds without taking any stamina damage). with the 50 monkeyman mask stack, this stamina damage suddenly becomes kind of busted, especially since it apparently can affect people a little more than a screen away from you (can this affect people through walls?).

I'd recommend making the effects of these masks not RNG-dependent, but also making the AoEs of multiple masks not stack with each other.

Reply:
I'm fine with the current effects as they are.

Reply:
Weren't we starting the process if they have a mind? Wouldn't this not matter?